edx_proctoring/static/proctoring/js/views/proctored_exam_view.js

```diff
@@ -71,6 +76,22 @@ var edx = edx || {};
                 "credit eligibility in this course.\n");
         },
         updateRemainingTime: function (self) {
+            self.timerTick ++;
+            if (self.timerTick % 5 == 0){
+                var url = self.model.url + '/' + self.model.get('attempt_id');
+                $.ajax(url).success(function(data) {
+                    if (data.status === 'error') {
+                        // Let the student know that his exam has failed due to an error.
+                        // This alert may or may not bring the browser window back to the
+                        // foreground (depending on browser as well as user settings)
+                        alert(gettext('Your exam has failed'));
```

Comment:
@catong can I get some appropriate messaging here.

This use case is that the end user is in an exam and we detect that the SoftwareSecure application was:

1) closed by the end-user
2) the SoftwareSecure application crashed abruptly

This is a little modal alert dialog box which appears. More informative text would be shown in the seq_proctored_exam_error.html template (below)